DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+		\hline
+		flux and associated parameters for a dipole model & PASS & several dipole flux values are present \\
+		\hline
+		parameters of an adaptive shape measurement and associated error & PASS & \texttt{base\_SdssShape\_*} \\
+		\hline
+	a measure of source extendedness & FAIL & not present in the catalog \\
+		\hline
+	the estimated background at the position of the object in the template image with associated uncertainty & FAIL & not present in the catalog \\
+		\hline
+	a measure of spuriousness & FAIL &  not present in the catalog \\
+		\hline
+	flags indicating problems encountered while computing the aforementioned attributes & PASS & many flags are present \\
+		\hline
+  calibrated flux and associated error for the DIAsource as measured on the Visit image & FAIL & not present in the catalog \\
+		\hline
+\end{tabular}
+	\caption{Required DIASource attributes for AG-00-20 \label{tab:diasrc}}
+\end{table}
+
+
+\textit{Faint DIASources satisfiying additional criteria are stored: FAILED}
+
+No specification for the ``additional criteria'' necessary to store 
+sub-threshold DIASources are available at this time; 
+this is a failure of requirements rather than of the pipelines.
+
+A new JIRA ticket, DM-12967, has been created to capture this issue.
+
+\textit{Relevant derived quantities are provided in pre-computed columns: PASSED}
+
+Examination of the DIASource record shows two entries,
+\texttt{ip\_diffim\_PsfDipoleFlux\_chi2dof} and
+\texttt{ip\_diffim\_DipoleFit\_chi2dof}, which provide goodness of fit measures
+as required.  Color-color diagrams are not relevant for single-visit DIASource
+catalogs. The requirement does not specify an exhaustive list of relevant
+derived quantities and hence cannot be fully tested as written.
+
 \subsection{AG-00-25}
 
-\appendix
+\textit{Measurements are Presented in Flux Units: FAILED}
+
+No measurements (or aggregated measurements, such as means) 
+were contained in the the DIAObject table as implemented at the 
+time of the test.
+
+\textit{Each DIAObject record contains required attributes: FAILED}
+
+From DMS-REQ-0271, the required attributes include: a unique identifier, 
+identifiers of the nearest stars and galaxies in the Object catalog, 
+a probability that the DIAObject is the Object, and a set of DIAObject 
+properties.
+
+We verified that all DIAObjects had a unique identifier by comparing the 
+output of \texttt{select count(distinct(id)) from dia\_objects;} and
+\texttt{select count(id) from dia\_objects;} on the Level 1 database.  
+Both returned 546688 objects.
+
+The attributes linking to the Object catalog are out of scope for this test, as
+they require joint processing with DRP.
+
+By examining the schema of the implemented Level 1 database, we determined that only one DIAObject property is implemented at this time, a count of the number
+of DIASources matched to it.  However, all values in the column were 1, which 
+is not expected for a real astrophysical dataset.
```

Comment:
I gather from @morriscb's comments in #ap-prototype-pipeline that he got to the bottom of this. I think the resulting ticket is DM-13052, right? If so, let's link to that.